vowpalwabbit/cbify.cc

```diff
@@ -109,6 +109,8 @@ void finish_cbify_reg(cbify_reg& data, std::ostream* trace_stream)
   if (trace_stream != nullptr) (*trace_stream) << "Max Cost=" << data.max_cost << std::endl;
 }
 
+VW_WARNING_STATE_PUSH
```

Comment:
could this warning be fixed with a rename instead ?

Reply:
yes that could be a fix, and I actually renamed the other case.
But since this is an init method, i thought maybe it makes sense to use the same name?
Instead of suppressing this warning, another way is perhaps to refactor using setter methods. WDYT?

Reply:
isn't this complaining about the function parameters i.e. `const std::size_t num_actions` and `std::size_t increment`? if so can't we just rename those to `num_actions_` and `increment_`?

Reply:
done, thanks